src/main/groovy/nextflow/executor/SimpleFileCopyStrategy.groovy

```diff
@@ -20,6 +20,7 @@
 
 package nextflow.executor
 import java.nio.file.Path
+import java.nio.file.Paths
```

Comment:
I don't think you really need to import it twice ;-)

Reply:
Hi @MaxUlysse, they're two different classes - `Path` and `Paths`. `Path` is actually an interface, with no constructor, so I use the `Paths` utilities to create `Path` objects.

Reply:
:man_facepalming: 
Oooh, read that definitively too fast...

Sorry, I was just looking at the PR, and I found it was strange having twice the same import...
It does make more sense this way...

Sorry again for my useless attempt at helping out...